src/Schema/Events/Click.ts

```diff
@@ -1583,3 +1583,27 @@ export interface ClickedOnMakeOfferCheckbox {
   context_page_owner_id?: string
   label: boolean
 }
+
+/**
+ * A Partner clicks on the Duplicate button for duplicating an artwork in the CMS.
+ *
+ * This schema describes events sent to Segment from [[clickedOnDuplicateArtwork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOnDuplicateArtwork",
+ *    context_module: "voltArtworksEdit",
+ *    original_artwork_id: "60de173a47476c000fd5c4cc"
+ *    duplicate_artwork_id: "6487822214765c000d34cbe8"
+ *    label: ["duplicate", "cancel", "open modal"]
+ *  }
+ * ```
+ */
+export interface ClickedOnDuplicateArtwork {
+  action: ActionType.clickedOnDuplicateArtwork
+  context_module: ContextModule
+  original_artwork_id: string
+  duplicate_artwork_id?: string
```

Comment:
Ah yes, interesting point about the `duplicate_artwork_id`. If we do truly want it, it the event will have to be fired after the response comes back from Gravity which kinda makes this event not really a "click" type event, right?